translate/src/modules/search/components/SearchBox.tsx

```diff
@@ -92,7 +104,29 @@ export function SearchBoxBase({
       }
       return next;
     },
-    { authors: [], extras: [], statuses: [], tags: [] },
+    {
+      authors: [],
+      extras: [],
+      statuses: [],
+      tags: [],
+    },
+  );
+
+  const [options, updateOptions] = useReducer(
```

Comment:
Should we be more explicit, and always prepend `option(s)` with `search`, so `searchOptions`, `updateSearchOptions`?